jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -74,6 +74,33 @@ def _warn_if_not_found(app, file_lists):
     return found
 
 
+def _parse_as_image(path):
```

Comment:
I wonder if you can take advantage of the Imviz parser that also handles ASDF/GWCS. It does not return stuff as CCDData but you can write a wrapper to do that part.

Reply:
Likely, but that will be another PR I think. This was just to get the data I had working at all so I can do other things!

Reply:
See #783